packages/node/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -457,6 +468,41 @@ function patchRequestToCaptureBody(req: IncomingMessage, isolationScope: Scope):
   }
 }
 
+/**
+ * Mutates the passed in `options` and adds `sentry-trace` / `baggage` headers, if they are not already set.
+ */
+function addSentryHeadersToRequestOptions(
+  url: string,
+  options: RequestOptions,
+  propagationDecisionMap: LRUMap<string, boolean>,
+): void {
+  // Manually add the trace headers, if it applies
+  // Note: We do not use `propagation.inject()` here, because our propagator relies on an active span
+  // Which we do not have in this case
+  const tracePropagationTargets = getClient()?.getOptions().tracePropagationTargets;
+  const addedHeaders = shouldPropagateTraceForUrl(url, tracePropagationTargets, propagationDecisionMap)
+    ? getTraceData()
+    : undefined;
+
+  if (!addedHeaders) {
+    return;
+  }
+
+  if (!options.headers) {
+    options.headers = {};
+  }
+  const headers = options.headers;
+
+  Object.entries(addedHeaders).forEach(([k, v]) => {
+    // We do not want to overwrite existing headers here
+    // If the core UndiciInstrumentation is registered, it will already have set the headers
+    // We do not want to add any then
```

Comment:
Do http and undici get applied to the same requests? Or did you mean
```suggestion
    // We do not want to overwrite existing headers here
    // If the core HttpInstrumentation is registered, it will already have set the headers
    // We do not want to add any then
```
?

Reply:
oops, copy pasted that and forgot to adjust! 😅